examples/predict.py

```diff
@@ -0,0 +1,21 @@
+import os
+
+###################################################
+# Assume the user already ran download_and_train.py
+###################################################
+
+gpu_id = 0
+
+image_save_dir = '{}/images/'.format(os.getcwd())
+model_save_dir = '{}/model/'.format(os.getcwd())
+prefs_save_path = '{}/prefs.json'.format(os.getcwd())
+
+data_save_path_test = '{}/image_list_test.csv'.format(image_save_dir)
+
+command_str = 'fnet predict ./ ' \
```

Comment:
Again, this implies your model is saved in the cwd. `model_save_dir` above is unused. This also means your data directory, `images`, is inside your model directory.